tools/autograd/derivatives.yaml

```diff
@@ -548,7 +548,7 @@
 - name: norm(Tensor self, Scalar p)
   self: norm_backward(grad, self, p, result)
 
-- name: norm(Tensor self, Scalar p, int64_t dim, bool keepdim)
```

Comment:
Yeah that's true for `dim`, not `keepdim=False` I suppose since it does not accept None. I think what was done before is that python_arg_parser parse these two signature:
`norm(Scalar p =2)`
`norm(Scalar?p, int64_t dim, bool keepdim=False`
for different calls, and these two have slightly different behaviors. I do think that we could merge these two and I will try to do it in a follow up PR.